src/Compilers/Core/Portable/Operations/IVariableDeclarationGroupOperation.cs

```diff
@@ -5,23 +5,28 @@
 namespace Microsoft.CodeAnalysis.Operations
 {
     /// <summary>
-    /// Represents an operation with one or more <see cref="IVariableDeclarationOperation"/>.
+    /// Represents a variable declaration statement.
+    /// </summary>
     /// <para>
-    /// Current usage:
-    ///  (1) C# local declaration statement.
-    ///  (2) VB Dim statement.
+    /// Current Usage:
+    ///   (1) C# local declaration statement
+    ///   (2) C# fixed statement
+    ///   (3) C# using statement
+    ///   (4) VB Dim statement
+    ///   (5) VB Using statement
     /// </para>
-    /// </summary>
     /// <remarks>
     /// This interface is reserved for implementation by its associated APIs. We reserve the right to
     /// change it in the future.
     /// </remarks>
-    public interface IVariableDeclarationsOperation : IOperation
+    public interface IVariableDeclarationGroupOperation : IOperation
     {
         /// <summary>
-        /// Variables declared.
+        /// Variable declaration in the statement.
         /// </summary>
+        /// <remarks>
+        /// In C#, this will always be a single declaration, with all variables in <see cref="IVariableDeclarationOperation.Declarators"/>.
+        /// </remarks>
```

Comment:
Can we omit the `IVariableDeclarationGroupOperation` in C#?

Reply:
The idea is to keep the operation trees consistent between the languages, so we want to keep the group.

---
In reply to: [148027581](https://github.com/dotnet/roslyn/pull/22842#discussion_r148027581) [](ancestors = 148027581)